### DISCUSSION TITLE
Can you confirm this?

### DISCUSSION THREAD

**Discussion by smah** [aFarkas/lazysizes#975] [Q&A]
Hi there,
I added the lazyload to the image tags. Saw the loading and then the actual image. I was shocked to see the code change (src, lazyloaded) when viewing the source code.

Source code
< img src="/assets/img/1px.png" data-src="/assets/img/comparison/esg-father-son1.jpg?v=20200927" class="lazyload" alt="ESG."  /> 

Desktop output
<img src="assets/img/comparison/esg-father-son1.jpg" data-src="assets/img/comparison/esg-father-son1.jpg?v=20200927" class= lazyloaded" alt="ESG">

It's correct. right?

**Comment by tinect** [comment]
This is correct, while this is the way lazysizes works.
